components/Heart.tsx

```diff
@@ -16,21 +16,27 @@ export default function Heart(props: {
 
   if (props.isLoading) {
     return (
-      <button className={styles.heart} onClick={handleClick}>
-        <Icon data={favorite_filled} />
-      </button>
+      <div>
+        <button className={styles.heart} onClick={handleClick}>
+          <Icon data={favorite_filled} />
+        </button>
+      </div>
```

Comment:
Why are you wrapping it in a div?

Reply:
I use a flexbox in the card header where I align the heart with the title.

Without wrapping the heart in a div it seems like "align-items: flex-start" does not affect the button, whereas it does affect the div.

Is it better practise to add the div in the card header component?

Reply:
I don't know. What do you think?

Why is the button behaving different than a div?

Could we change that behavior of the button to reflect what you described? [Maybe helpful](https://developer.mozilla.org/en-US/docs/Web/CSS/display)